model/src/main/java/com/flipkart/flux/domain/Event.java

```diff
@@ -48,7 +48,7 @@
     private EventStatus status;
 
     /** Instance Id of state machine with which this event is associated */
-    private Long stateMachineInstanceId;
+    private String stateMachineInstanceId;
```

Comment:
Actually we can avoid autoincrement id for this table by having <stateMachineInstanceId, name> composite key